src/block/card/style.js

```diff
@@ -78,7 +82,21 @@ export const createStyles = props => {
 		...createImageBackgroundStyleSet( 'image%s', 'ugb-card__image', props.attributes ),
 	} )
 	if ( show.imageHeight ) {
-		styles.push( ...createResponsiveStyles( '.ugb-card__image', 'imageBackground%sHeight', 'height', '%spx', props.attributes, true ) )
+		styles.push( {
```

Comment:
Better to put the definition of `maxResponsiveSize` here so that it's near the area where it will be used